spec/factories/topics.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_topics_on_name  (name) UNIQUE
+#
+FactoryBot.define do
+  factory :topic do
+    name { Faker::Lorem.unique.word }
```

Comment:
Uniqueness is something that Faker manages poorly in a performance sense. Your tests will slowly start to take more time to run until you find yourself in a place where you don't know why it happens. Anyway, just a heads up so it doesn't happen to you. No need to change it here but I generally use sequence for that exact reason.